l3kernel/testfiles-l3doc/github-1398.lvt

```diff
@@ -0,0 +1,25 @@
+%
+% Copyright (C) The LaTeX Project
+%
+
+\input regression-test.tex
+\documentclass{l3doc}
+
+\begin{document}
+\begin{function}[noTF]{\clist_get:NN}
+  No variant.
+\end{function}
+
+\begin{function}[noTF]{\clist_get_b:NN, \clist_get_b:cN}
+  Single variant.
+\end{function}
+
+\begin{function}[noTF]{\clist_get_c:NN, \clist_get_c:cN, \clist_get_c:Nc}
+  Multiple variants.
+\end{function}
+
+\START
+\showoutput
+\clearpage
+\OMIT
```

Comment:
I don't like testing with `\showoutput` which dumps a lot of less readable log lines, but it seems there's no better way.